.github/workflows/all.yml

```diff
@@ -28,6 +28,4 @@ jobs:
           set -x
           which go
           go version
-          export GOMAXPROCS=2
-          args=("-p" "2" "-race")
```

Comment:
This two lines are likely still good idea given how github actions reports the wrong number of (available) cores and then test fail due to that.

Reply:
I think the last time when I dealt with that, I found that it's no longer the case (that GH actions reported the wrong number of cores)

Reply:
Anyway, dropped the change for now